File: media.js
const assert = require('assert');

const media = function(n1, n2, n3) {
  let media = (n1 + n2 + n3) / 3;
  let resultado = parseInt(media);
  return resultado;
}

try {
  // console.log(media(7,8,7))
  assert.equal(5, media(7,8,0));
} catch(e) {
  console.log(e);
}

File: C/media.c
#include <stdio.h>
#include <assert.h>

int media(int n1, int n2, int n3)
{
  double media = (int) (n1 + n2 + n3) / 3;
  return media;
}

int main()
{
  assert(8 == media(8,9,8));
  return 0;
}